### PR TITLE
Convert Torpedo to use go modules

### PR DESCRIPTION
**Pull request by rodrigc**
This change is similar to what was done for sched-ops here: https://github.com/portworx/sched-ops/pull/154

### CODE REVIEW

**Comment by thiguetta** [comment]
do we have it tested somewhere? post the link here

File: Makefile [hunk 1]
```diff
@@ -48,24 +57,24 @@ build:
...
 	ginkgo build -r
 
 	find . -name '*.test' | awk '{cmd="cp  "$$1"  $(BIN)"; system(cmd)}'
 	chmod -R 755 bin/*
 
 vendor-update:
-	dep ensure -update
+	go mod download
 
 vendor:
```
Comment (harsh-px) [inline Makefile]: add vendor tidy

Comment (rodrigc) [reply Makefile]: added

**Comment by rodrigc** [comment]
@thiguetta tested here: https://jenkins.portworx.co/job/Torpedo/job/tp-dev-test/280/parameters/

**Pull request merged**